keyboards/manta60/rules.mk

```
@@ -0,0 +1,34 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes       # Virtual DIP switch configuration
```

Well, I'd recommend Bootmagic Lite, since it's less prone to unexpected behavior.
```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```